.github/workflows/deploy-wf-service.yml

```
@@ -101,39 +128,35 @@ jobs:
+            docker tag ${{ env.REGISTRY }}/${{ env.IMAGE_NAME}}:${{ env.SHORT_HASH }}-dev ${{ env.REGISTRY }}/${{ env.IMAGE_NAME}}:${{ inputs.environment }}
           fi
           docker images
-          # docker push ${{ env.REGISTRY }}/${{ env.IMAGE_NAME}}:${{ inputs.environment }}
+          docker push ${{ env.REGISTRY }}/${{ env.IMAGE_NAME}}:${{ env.SHORT_HASH }}-${{ inputs.environment }}
+          docker push ${{ env.REGISTRY }}/${{ env.IMAGE_NAME}}:${{ inputs.environment }}
 
   build-and-push-ee-image:
     runs-on: ubuntu-latest
     needs: [check_if_data_migration_needed,tag-and-push-image]
-    if: ${{(needs.check_if_data_migration_needed.outputs.should_build == 'true')}}
+    if: always() && ${{(needs.check_if_data_migration_needed.outputs.should_build == 'true')}}
```

actionlint

if: condition "always() && ${{(needs.check_if_data_migration_needed.outputs.should_build == 'true')}}" is always evaluated to true because extra characters are around ${{ }}

(if-cond)